init/init.go

```diff
@@ -168,10 +171,21 @@ func getLaunchConfig(cfg *config.CloudConfig, dockerCfg *config.DockerConfig) (*
 	return &launchConfig, args
 }
 
+func isInitrd() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs("/", &stat)
+	return stat.Type == 0x01021994
```

Comment:
Well shoot, not sure what is the right incarnation here.  You need to set the constant to a uint32 I just don't know what happens when you cast it.